docs/design/architectural-overview.md

```diff
@@ -0,0 +1,168 @@
+# Kuadrant Architectural Overview [Draft]
```

Comment:
Nowhere is described the effective (HHTP) flow of a request to a gateway configured by Kuadrant (showing the multi cluster aspect, rate limiting etc). That projection / view could be useful.

Reply:
hmm so perhaps this image is not clear enough or we need a separate one entirely https://github.com/Kuadrant/architecture/pull/24/files#diff-0997a2bfd6775f89fe2228aabc6093a9a547c746f1e778e5731da5e4563287e1